tests/com/magento/idea/magento2plugin/actions/generation/generator/CLICommandDiXmlGeneratorTest.java

```diff
@@ -13,22 +13,27 @@
 import com.magento.idea.magento2plugin.magento.packages.Package;
 import com.magento.idea.magento2plugin.util.CamelCaseToSnakeCase;
 
+@SuppressWarnings({
+        "PMD.FieldNamingConventions",
+        "PMD.LongVariable",
```

Comment:
Let's suppress the `LongVariable ` on a global level. I believe variable names should be descriptive and self-explanatory.